spidermon/contrib/scrapy/extensions.py

```diff
@@ -62,13 +67,16 @@ def from_crawler(cls, crawler):
             crawler=crawler,
             spider_opened_suites=crawler.settings.getlist('SPIDERMON_SPIDER_OPEN_MONITORS'),
             spider_closed_suites=crawler.settings.getlist('SPIDERMON_SPIDER_CLOSE_MONITORS'),
+            engine_stopped_suites=crawler.settings.getlist('SPIDERMON_ENGINE_STOP_MONITORS'),
             spider_opened_expression_suites=crawler.settings.getlist('SPIDERMON_SPIDER_OPEN_EXPRESSION_MONITORS'),
             spider_closed_expression_suites=crawler.settings.getlist('SPIDERMON_SPIDER_CLOSE_EXPRESSION_MONITORS'),
+            engine_stopped_expression_suites=crawler.settings.getlist('SPIDERMON_ENGINE_STOP_EXPRESSION_MONITORS'),
             expressions_monitor_class=crawler.settings.get('SPIDERMON_EXPRESSIONS_MONITOR_CLASS'),
             periodic_suites=crawler.settings.getdict('SPIDERMON_PERIODIC_MONITORS'),
         )
         crawler.signals.connect(ext.spider_opened, signal=signals.spider_opened)
         crawler.signals.connect(ext.spider_closed, signal=signals.spider_closed)
+        crawler.signals.connect(ext.engine_stopped, signal=signals.engine_stopped)
```

Comment:
This line could be tested like:
https://github.com/scrapinghub/spidermon/pull/120

Reply:
Nothing against the approach, I just think that we only need to test if it _connected_ to the signal.
The test of being called when the signal is triggered, is already handled by scrapy.

Reply:
But I think it's best to test this on your PR  right? or on this if yours get approved first.